### PR TITLE
fix: vm assume too many rejections

### PR DESCRIPTION
**Pull request by stevennevins**

### CODE REVIEW

File: test/unit/BLSApkRegistryUnit.t.sol [hunk 1]
```diff
@@ -800,8 +990,8 @@ contract BLSApkRegistryUnitTests_quorumApkUpdates is BLSApkRegistryUnitTests {
         uint256 blockGap,
         uint256 randSeed
     ) external {
-        cheats.assume(numRegistrants > 0 && numRegistrants < 100);
-        cheats.assume(blockGap < 100);
+        numRegistrants = bound(numRegistrants, 1, 100);
+        blockGap = bound(blockGap, 0, 10);
```
Comment (ChaoticWalrus) [inline test/unit/BLSApkRegistryUnit.t.sol]: any reason for switching the upper bound here to 10 from 100?
seems fine either way tho

File: test/unit/BLSApkRegistryUnit.t.sol [hunk 2]
```diff
@@ -855,7 +1054,7 @@ contract BLSApkRegistryUnitTests_quorumApkUpdates is BLSApkRegistryUnitTests {
         uint32 wrongBlockNumber,
         uint256 randSeed
     ) external {
-        cheats.assume(numRegistrants > 0 && numRegistrants < 100);
+        numRegistrants = bound(numRegistrants, 1, 100);
```
Comment (ChaoticWalrus) [inline test/unit/BLSApkRegistryUnit.t.sol:1057]: only other change I can identify; LGTM

**Review by ChaoticWalrus — APPROVED** [review]
Looks fine. I commented on the two actual changes I see -- everything else is formatting.

File: test/unit/BLSApkRegistryUnit.t.sol (hunk 1 above)
Comment (stevennevins) [reply test/unit/BLSApkRegistryUnit.t.sol]: nice catch that was a typo

Comment (stevennevins) [reply test/unit/BLSApkRegistryUnit.t.sol]: [2bd61f3](https://github.com/Layr-Labs/eigenlayer-middleware/pull/247/commits/2bd61f30ac0fef0bd91a8caedca26b1d31facdda)

**Pull request merged**